src/visualization/components/internal/AdaptiveZoomOption.tsx

```diff
@@ -0,0 +1,72 @@
+// Libraries
+import React, {CSSProperties, FC} from 'react'
+
+// Components
+import {
+  AlignItems,
+  ComponentSize,
+  FlexBox,
+  FlexDirection,
+  InfluxColors,
+  InputLabel,
+  QuestionMarkTooltip,
+  SlideToggle,
+} from '@influxdata/clockface'
+
+// Metrics
+import {event} from 'src/cloud/utils/reporting'
+
+interface AdaptiveZoomToggleProps {
+  adaptiveZoomHide: boolean
+  update: (obj: any) => void
+  testID?: string
+  type: string
+}
+
+const getToggleColor = (toggle: boolean): CSSProperties => {
+  if (toggle) {
+    return {color: InfluxColors.Grey95}
+  }
+  return {color: InfluxColors.Grey65}
+}
+
+export const AdaptiveZoomToggle: FC<AdaptiveZoomToggleProps> = ({
+  adaptiveZoomHide,
+  update,
+  testID = 'adaptive-zoom-toggle',
```

Comment:
does this need to be changeable? It looks like the only place this is used is here, and it's not changing the default test id. YAGNI